### PR TITLE
Delete thread pool metrics

### PR DESCRIPTION
**Pull request by laurit**

### CODE REVIEW

**Review by theletterf — APPROVED** [review]
Will this update the metadata.yaml file?

**Comment by laurit** [comment]
> Will this update the metadata.yaml file?

yes

File: docs/metrics.md [hunk 1]
```diff
@@ -7,8 +7,6 @@
 
 The Splunk Distribution of OpenTelemetry Java agent gathers basic application metrics.
 Because these splunk-specific metrics are still experimental they are not enabled by default.
```
Comment (breedx-splk) [inline docs/metrics.md:9]: After this are there any more remaining Splunk-specific metrics left?

**Review by breedx-splk — APPROVED** [review]

File: docs/metrics.md (hunk 1 above)
Comment (laurit) [reply docs/metrics.md:9]: There are profiling metrics.

**Pull request merged**